src/Symfony/Bridge/ProxyManager/README.md

```diff
@@ -3,6 +3,7 @@ ProxyManager Bridge
 
 The ProxyManager bridge provides integration for [ProxyManager][1] with various
 Symfony components.
+The project is discontinued; 6.4 is the last version.
```

Comment:
let's add a blank line before to make  a new §

Reply:
Sure, I made it bold too.